### PR TITLE
Support disabling insecure ports

### PR DESCRIPTION
**Pull request by sophokles73**
This addresses #457

Do not expose insecure service port if insecure port is disabled for a component. Support disabling the insecure port of the Dispatch Router of the AMQP 1.0 based example messaging infrastructure.

### CODE REVIEW

**Comment by sophokles73** [comment]
@calohmn would you mind taking a look?

File: charts/hono/config/router/qdrouterd.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "saslPlugin": "Hono Auth"
   }],
 
+{{- if .Values.amqpMessagingNetworkExample.insecurePortEnabled }}
```
Comment (calohmn) [inline charts/hono/config/router/qdrouterd.json:39]: This condition can also be placed around the definition of the `amqp` port 5672 in `dispatch-router-deployment.yaml`.

File: charts/hono/values.yaml [hunk 2]
```diff
@@ -350,7 +350,7 @@ adapters:
       # exposed AMQP endpoints.
       # If not set, the adapter by default exposes the secure and insecure ports
       # using an example key and certificate.
-      amqp:
+      amqp: {}
```
Comment (calohmn) [inline charts/hono/values.yaml:353]: For consistency, this change could also be made to the `authServer.hono.auth.amqp` field below, adapting the `hono-service-auth-secret.yaml` accordingly.

**Review by calohmn — APPROVED** [review]
LGTM

**Pull request merged**